site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -19,6 +19,18 @@ True if CleanSpeak username moderation is enabled.
 [field]#application.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Application that should be persisted.
 
+[field]#application.emailConfiguration.emailVerificationEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.19.0#::
+The Id of the Email Template used to send emails to users to verify that their email address is valid.
+
+[field]#application.emailConfiguration.forgotPasswordEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.19.0#::
+The Id of the Email Template that is used when a user is sent a forgot password email.
+
+[field]#application.emailConfiguration.passwordlessEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.19.0#::
+The Id of the Passwordless Email Template.
+
+[field]#application.emailConfiguration.setPasswordEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.19.0#::
+The Id of the Email Template that is used when a user had their account created for them and they must set their password manually and they are sent an email to set their password.
+
```

Comment:
Wow... good catch, this was probably my bug. 

These fields are optional, and when configured, they will take precedence over the same configuration from the tenant when an application context is known.

Reply:
ok, will add that.